File: src/WebUI/ClientApp/src/app/core/movie.ts
export class Movie {
  id: number;
  name: string;
  description: string;
  duration: number;
  constructor(id: number=0, name: string = '', description: string ='', duration: number = 0) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.duration = duration;
  }
}

File: src/Domain/Entities/Movie.cs
using demo_ca_app.Domain.Common;

namespace demo_ca_app.Domain.Entities
{
    public class Movie : AuditableEntity
    {
        public int Id { get; set; }

        public string Name { get; set; }
        public string Description { get; set; }
        public int Duration { get; set; } // minute
    }
}
